plaid/api/sandbox.py

```diff
@@ -15,6 +15,23 @@ def reset_login(self, access_token):
         })
 
 
+class PublicToken(API):
+    '''Sandbox public token endpoints.'''
+
+    def create(self, institution_id, initial_products):
+        '''
+        Generate a public token for sandbox testing.
+
+        :param  str     institution_id:
+
+        :param  [str]   initial_products:
+        '''
+        return self.client.post_public_key('/sandbox/public_token/create', {
+            'institution_id': institution_id,
+            'initial_products': initial_products
+        })
```

Comment:
@evanlimanto this endpoint can take a options param too, right now only a single support value:

```
options.webhook String
```

Reply:
fixed!